py/map.c

```diff
@@ -167,134 +173,139 @@ mp_map_elem_t *mp_map_lookup(mp_map_t *map, mp_obj_t index, mp_map_lookup_kind_t
     }
 
     // if the map is an ordered array then we must do a brute force linear search
+    #if !MICROPY_PY_MAP_ORDERED
     if (map->is_ordered) {
-        for (mp_map_elem_t *elem = &map->table[0], *top = &map->table[map->used]; elem < top; elem++) {
```

Comment:
Is this uncrustify which wants to dedent all the code?  It's not great.  Would it fix it to add an extra set of `{  }` around this code, outside the #if?

Reply:
Yep these were both from uncrustify breaking the build. I'll try the `{ }`

Reply:
Good call, yes the brackets appear to have fixed the formatting.